packages/terra-date-input/src/terra-dev-site/test/date-input/InvalidDateInputField.test.jsx

```diff
@@ -4,8 +4,8 @@ import React, { useState } from 'react';
 import DateInputField from 'terra-date-input/lib/DateInputField';
 
 const Example = () => {
-  const [value, setValue] = useState('$5%g-ad-!f');
-  const [isInvalid, setIsInvalid] = useState(false);
+  const [value, setValue] = useState('1999-03-10');
```

Comment:
This test is intended to test passing down invalid value. What's the reason for changing to a valid value?

Reply:
I renamed the test for this to [InvalidProvidedInputField](https://github.com/cerner/terra-framework/pull/923/files/25d76f83c6c7e1986883293c7b219aad195e5f7c#diff-952c2da42252c1edf25365b7ce031dd5R1) since its a test of what happens when the component is provided an invalid date. I am now using this test's name to signify a test of the `isInvalid` css styling.

Currently InvalidProvidedInputField test is disabled as the contrast ratio for the placeholder is causing the test to fail